src/store/actions.js

```diff
@@ -96,6 +96,22 @@ export default {
       }
     }
   },
+  async getTokensPublicInfo({ state: { nextTokensFetch, current }, commit }) {
```

Comment:
why not fetch from backend? the fetch logic should probably not be in the store actions itself